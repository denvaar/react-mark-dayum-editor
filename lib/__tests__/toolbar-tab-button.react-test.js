import React from 'react';
import renderer from 'react-test-renderer';

import ToolbarTabButton from '../components/toolbar-tab-button';


it('renders correctly in active state', () => {
  const tree = renderer.create(
    <ToolbarTabButton label={"Write"} activeIf={true} onClick={null} />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});

it('renders correctly in inactive state', () => {
  const tree = renderer.create(
    <ToolbarTabButton label={"Write"} activeIf={false} onClick={null} />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
