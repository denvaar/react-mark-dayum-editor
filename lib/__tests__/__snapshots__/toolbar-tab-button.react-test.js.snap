// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly in active state 1`] = `
<button
  className="md-toolbar-tablinks active"
  onClick={null}
  type="button"
>
  Write
</button>
`;

exports[`renders correctly in inactive state 1`] = `
<button
  className="md-toolbar-tablinks"
  onClick={null}
  type="button"
>
  Write
</button>
`;
